APE0.rst

```diff
@@ -0,0 +1,393 @@
+to work towards consensus whenever practical. The Coordination Committee
+should look for ways to use its authority as little as possible, striving
+to involve the community in decisions whenever possible, and delegating
+responsibilities.
+
+When delegating its authority to create roles or make appointments, the
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its authority, the Coordination Committee must first attempt to reach
+a consensus. If a consensus cannot be reached the Committee votes. Every
+Coordination Committee member must either vote or explicitly abstain. Members
+with a conflict of interest on a particular vote must abstain. The final
+decision regarding what constitutes a conflict of interest is determined by the
+Ombudsperson in accordance with the conflict of interest policy. Passing
+decisions require a strict majority of non-abstaining Coordination Committee
+members.
+
+Whenever possible, the Coordination Committee's deliberations shall be held in
+public in a manner that permits discussion and comments from the community.
+Final decisions made by the Coordination Committee will be published in a
+publicly visible location.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats.
+Notice of the election must be posted via the Project’s standard communication
+channels (reaching all active members) at least one month before the election
+begins. The election consists of two phases, each lasting two weeks:
+
+* Phase 1, Nominations: Candidates will be nominated and confirm their interest
+  in serving. Candidates can be nominated by anyone, including themselves, and
+  do not have to be `Voting Members <votingmembers>`_ to be nominated.
+  Candidates should provide a brief statement in support of their candidacy.
+* Phase 2, Voting: Each `Voting Member <votingmembers>`_ can vote for zero or
+  more of the candidates, up to the number of candidates. Voting is performed
+  anonymously. Candidates are ranked by the total number of votes they receive.
+  If a tie occurs, it may be resolved by mutual agreement among the candidates,
+  or else the winner will be chosen at random.
+
+The election process is managed by a non-voting Returns Officer nominated by
+the outgoing Coordination Committee, although the process (specific choice of
+web site, etc) can be set up by the Coordination Committee or their delegates.
+For the initial election, the Returns Officer will be nominated by the NumFOCUS
+Executive Officer. Elections will be carried out every year, provided that at
+least one of the Coordination Committee members has reached the end of their
+term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when
+the election results are finalized, except when the election was to fill a
```

Comment:
> except when the election was to fill a
vacancy created partway through a term.

why there is this distinction for having a second-class citizen election and "normal" ones? A system where the two longest sitting members are up for reelection seems much fairer.  There is no reasoning documented in the running notes about this.

Reply:
I am not positive, but I think this so that 1) the terms of everyone else on the CoCo don't change just because there is a vacancy and 2) the terms stay staggered. Otherwise, if elections were based on relative time served, if there was an unexpected vacancy, everyone else's terms on the CoCo would change. You could not change their terms, but then you'd end up filling two seats at once rather than one seat at a time and losing the desired continuity. I think this is mostly following the US model of "Special Elections" to fill vacant House and Senate seats: https://en.wikipedia.org/wiki/List_of_special_elections_to_the_United_States_House_of_Representatives

Reply:
Yes, @kelle's explanation is my recollection as well. If we want fixed-length terms this is the only way to ensure that staggering continues perpetually.

I didn't remember until I looked it up but I see it in the WG notes under "Restagger term length if it gets de-synced." - it looks like there was a vote unanimous with one abstention with this general idea, and then it was noted specifically as a change that went into the final vote to approve the document.

Reply:
Should all our things be based on the US? Naive question I know, as many of you apparently think yes is the only acceptable answer.
E.g. having an early election doesn't mean a new government has a shorter term than normal.

I here just simply wonder, why someone voted in, in an otherwise fully valid election has shorter terms, just because they are filling in the place of someone else. It will be filling one or two seats yearly anyways.

Otherwise, this document should call it clearly a by-election if it has different terms and consequences.

Reply:
The US might be the most prominent place, but it's far from the only one. If e.g. someone replaces a member of parliament in Germany, the don't get a full term, they get to stay in until the next general election, even if that's only a few weeks. Similarly, in the charity that I volunteered in in high school, most (but not all) elected officers had staggered terms and if someone was replaced mid-term, they would only get to serve a half-term to keep the phasing. So it's actually quite common to do it this way in my experience.

The wording in APE0 is explicit about this, there is no ambiguity, and I don't see a problem. As a non-native speaker, I can't say if this should be called a "by-election" instead of an "election", but I can say that German uses the same word for both cases.

Reply:
Then refer to the process, by-election, or special election rather than to say we are to follow how things are done in the US senate. Quite patronising.

Other language(s) also doesn't use the same phrase for both cases, in case we start counting examples.

Reply:
It also seems to be how seats in the House of Commons are filled, should an MP resign or die in the middle of their term. I just learned that from wikipedia after I googled "by-election", a term that I did not know before. Anyway, either American or British English is fine for this (while German is not - we don't actually vote in this case, the replacement for a member or parlament is just the highest person on the list from the original election who did not yet get a seat).

Reply:
yes by-elections have different terms and conditions than normal elections. So, if it has to be a by-election then call it by-election, or if it's a normal election have the normal length of term come with it. 
Examples can be brought for both approaches from many countries.

I originally wondered why it is better for us to not to have a full term, and hoped to have people chime in who made the unanimous decision, as there was no reasoning captured in the running notes. I don't mean to contest the decision, but interested in why it's better. 

But I would contest the phrasing, and ask to call it by-election if the current ruling stands.

Reply:
I'm doing a hit and run comment, so to speak. In reviewing all these references to how countries handle special elections, I'll note that the motive there is not to maintain continuity, but rather to keep regular elections synchronized on a national level. So in this case perhaps some different mechanism can be considered. For example, if the term is less than half over, a special election will be held, that will only run to the end of the regular term, and if it is more than half over, it will run to the end of the next term. There seems little point in having an election for someone that will only spend a few months in the position. For what it is worth.

Reply:
> I originally wondered why it is better for us to not to have a full term, and hoped to have people chime in who made the unanimous decision, as there was no reasoning captured in the running notes. I don't mean to contest the decision, but interested in why it's better.

@bsipocz the reasoning is that we would want to avoiding ending up in a situation where all CoCo members have terms that are in sync which would then mean no elections some years (for example if two CoCo members have elections coming up and the two others stand down just around that time). I'd be in favour of using the term by-elections here to make it clear this is a different process. I also agree with Perry that we don't necessarily want to hold elections for someone who will hold the position for 2 months, though I'm not really sure where to place the cutoff.

Reply:
This has been changed to say "by-election" and clarify some of the wording a bit to be very concrete about this.